packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.tsx

```diff
@@ -102,34 +113,37 @@ export class MessageBar extends BaseComponent<IMessageBarProps, IMessageBarState
   private _renderMultiLine(): React.ReactElement<React.HTMLProps<HTMLAreaElement>> {
     return (
       <div
-        className={ this._getClassName() + ' ms-MessageBar-multiline ' + styles.multiline }
+        className={ this._getClassName() + ' ms-MessageBar-multiline ' + styles.multiline + ' ' +
+          css({
+            [styles.dismissalMultiline]: this.props.onDismiss != null,
+            [styles.actionableMultiline]: this.props.actions != null
+          })
+        }
         role='status'
         aria-live={ this._getAnnouncementPriority() }>
-        <div className={ css('ms-MessageBar-content', styles.content) }>
+        <div className={ css(styles.content) }>
           { this._getIconSpan() }
-          <div className={ css('ms-MessageBar-actionables', styles.actionables) }>
-            { this._renderInnerText() }
-            { this._getActionsDiv() }
-            { this._getDismissDiv() }
-          </div>
+          { this._renderInnerText() }
+          { this._getDismissDiv() }
         </div>
+        { this._getActionsDiv() }
       </div>
     );
   }
 
   private _renderSingleLine(): React.ReactElement<React.HTMLProps<HTMLAreaElement>> {
     return (
-      <div className={ this._getClassName() + ' ms-MessageBar-singleline ' + styles.singleline }>
-        <div className={ css('ms-MessageBar-content', styles.content) }>
+      <div className={ this._getClassName() + ' ms-MessageBar-singleline ' + styles.singleline + ' ' +
+        css({
+          [styles.dismissalOneline]: this.props.onDismiss != null,
+          [styles.actionableOneline]: this.props.actions != null
+        }) }>
+        <div className={ css(styles.content) }>
```

Comment:
...but this is a misuse of the css helper. why not pass in styles.content directly?